metricbeat/module/rabbitmq/url.go

```diff
@@ -0,0 +1,26 @@
+package rabbitmq
+
+import (
+	"github.com/elastic/beats/metricbeat/mb/parse"
+)
+
+// Subpaths to management plugin endpoints
+const (
+	ConnectionsPath = "/api/connections"
+	ExchangesPath   = "/api/exchanges"
+	NodesPath       = "/api/nodes"
+	QueuesPath      = "/api/queues"
+)
+
+const (
+	defaultScheme = "http"
+	pathConfigKey = "management_path_prefix"
+)
+
+var (
+	// HostParser parses host urls for RabbitMQ management plugin
+	HostParser = parse.URLHostParserBuilder{
```

Comment:
I wonder if it could become an issue that all Metricsets will now share one host parser as a global object?

Reply:
It shouldn't be an issue.